### PR TITLE
Fixes #25243 - option interface-mtu parsed correctly

### PR DESCRIPTION
**Pull request by lzap**
It was erroring out when this was present in a conf file:

```
subnet 10.0.0.0 netmask 255.255.255.0 {
    option interface-mtu 9000;
}
```

### CODE REVIEW

File: test/dhcp/dhcp_isc_subnet_service_initialization_test.rb [hunk 1]
```diff
@@ -70,6 +71,7 @@ def test_managed_subnets_network_addresses
     @initialization.load_configuration_file(DHCPD_CONFIG)
     subnets =  @subnet_service.all_subnets
     assert_equal "192.168.122.0", subnets[0].network
+    assert_equal 9000, subnets[0].options[:interface_mtu]
```
Comment (timogoebel) [inline test/dhcp/dhcp_isc_subnet_service_initialization_test.rb]: This does not match the test description, how about moving this to a separate test `test_managed_subnets_options`?

Comment (lzap) [reply test/dhcp/dhcp_isc_subnet_service_initialization_test.rb]: Sure, done.

**Review by timogoebel — APPROVED** [review]

**Comment by timogoebel** [comment]
Thanks, @lzap.

**Comment by tbrisker** [comment]
1.20 - 59fe1eb

**Pull request merged**